sys/net/gnrc/routing/rpl/gnrc_rpl_dodag.c

```diff
@@ -116,6 +116,8 @@ bool gnrc_rpl_instance_remove(gnrc_rpl_instance_t *inst)
 #endif
     gnrc_rpl_dodag_remove_all_parents(dodag);
     trickle_stop(&dodag->trickle);
+    evtimer_del(&gnrc_rpl_evtimer, (evtimer_event_t *)&dodag->dao_event);
```

Comment:
Can't the DODAG still be a member of another instance?

Reply:
Nope, the dodag struct is part of the instance struct  as a nested struct. It's not a pointer

Reply:
Arghs, sorry. I've somehow thought it was a pointer.